tensorflow/inference/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,174 @@
+    libcublas-dev-11-0=11.2.0.252-1 \
+    libcudnn8=${CUDNN_VERSION} \
+    libcudnn8-dev=${CUDNN_VERSION} \
+    libnccl2=${NCCL_VERSION} \
+    libnccl-dev=${NCCL_VERSION}  \
+    libgomp1 \
+    libffi-dev \
+    curl \
+    emacs \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev \
+    openssl \
+    libssl1.1 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
+ && apt-get update \
+ && apt remove -y --purge openssl \
+ && rm -rf /usr/include/openssl \
+ && apt-get install -y \
+    ca-certificates \
+ && tar -xzvf openssl-${OPENSSL_VERSION}.tar.gz \
+ && cd openssl-${OPENSSL_VERSION} \
+ && ./config && make && make test \
+ && make install \
+ && ldconfig \
+ && cd .. && rm -rf openssl-*
+
+# when we remove previous openssl, the ca-certificates pkgs and its symlinks gets deleted
+# causing sslcertverificationerror the below steps are to fix that
+RUN ln -s /etc/ssl/certs/*.* /usr/local/ssl/certs/
+
+# Install python3.7
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure && make && make install \
+ && make && make install && rm -rf ../Python-$PYTHON_VERSION*
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+# The 'apt-get install' of nvinfer-runtime-trt-repo-ubuntu1804-4.0.1-ga-cuda10.0
+# adds a new list which contains libnvinfer library, so it needs another
+# 'apt-get update' to retrieve that list before it can actually install the
+# library.
+# We don't install libnvinfer-dev since we don't need to build against TensorRT,
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update \
+# nvinfer-runtime-trt-repo doesn't have a 1804-cuda10.1 version yet. see:
+# https://developer.download.nvidia.cn/compute/machine-learning/repos/ubuntu1804/x86_64/
+ && apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated  \
+    nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
+ && apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated  \
+    libnvinfer7=7.1.3-1+cuda11.0 \
+    libnvinfer-plugin7=7.1.3-1+cuda11.0 \
+ && rm -rf /var/lib/apt/lists/*
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
+ && ln -s $(which ${PIP}) /usr/bin/pip
+
+# nginx + njs
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# Install openssl and libssl to resolve https://ubuntu.com/security/CVE-2020-1971
```

Comment:
is this not triaged by the above installation from source for version `1.1.1g`?

Reply:
The point to install openssl and libssl1.1.1 is to solve this issue